api/server/server.go

```diff
@@ -270,7 +274,9 @@ func createGRPCServer(opts []grpc.ServerOption, webProxyAddr string) (*grpc.Serv
 		grpcweb.WithOriginFunc(func(origin string) bool {
 			return true
 		}),
+		grpcweb.WithAllowedRequestHeaders([]string{"*"}),
```

Comment:
I think this is the default behavior, but just being explicit to remember exactly what we're doing here.